modular_zubbers/code/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -1,3 +1,249 @@
+
+/datum/job/atmospheric_technician/New()
+	.=..()
+	alt_titles += list(
+		"Pipe Cleaner",
+		"Pipe Inspector",
+		"Pipe Technician",
+	)
+
+/datum/job/bitrunner/New()
+	.=..()
+	alt_titles += list(
+		"Netrunner",
+		"Matrix Traversal Technician",
+	)
+
+/datum/job/blueshield/New()
+	.=..()
+	alt_titles += list(
+		"Blueshit",
+		"Chief Steward",
+		"Command Enforcer",
+		"Command Goon",
+		"Command Henchman",
+		"Steward",
+	)
+
+/datum/job/botanist/New()
+	.=..()
+	alt_titles += list(
+		"Apiarist",
+		"Apiculturist",
+		"Beekeeper",
+		"Beemaster",
+		"Farmer",
+		"Sodbuster",
+	)
+
+/datum/job/bouncer/New()
+	.=..()
+	alt_titles += list(
+		"Hired Muscle",
+	)
+
+/datum/job/captain/New()
+	.=..()
+	alt_titles += list(
+		"Cap'n",
+		"Chief Mate",
+		"Offworld Installation Manager",
+		"Premier",
+		"Staff Captain",
+		"Station Admiral",
+	)
+
+/datum/job/cargo_technician/New()
+	.=..()
+	alt_titles += list(
+		"Cargo Technician",
+		"Deckhand",
+		"Deck Rating",
+	)
+
+/datum/job/chaplain/New()
+	.=..()
+	alt_titles += list(
+		"Moffpriest",
+	)
+
+/datum/job/chemist/New()
+	.=..()
+	alt_titles += list(
+		"Alchemist",
+		"Explosives Specialist",
+		"Harmacist",
+		"Harmacologist",
+		"Plumbing Technician",
+		"Plumber",
+		"Pyrotechnics Specialist",
+		"Stimulant Manufacturing Technician",
+	)
+
+/datum/job/chief_medical_officer/New()
+	.=..()
+	alt_titles += list(
+		"Chief Boo-Boo Kisser",
+	)
+
+/datum/job/cook/New()
+	.=..()
+	alt_titles += list(
+		"Baker",
+		"Confectionist",
+		"Pastry Chef",
+		"Pâtissier",
+	)
+
+/datum/job/cyborg/New()
+	.=..()
+	alt_titles += list(
+		"Corporate Appliance",
+		"Droid",
+		"Gyndroid",
+		"Stationbound",
+		"Toaster",
+	)
+
+/datum/job/doctor/New()
+	.=..()
+	alt_titles += list(
+		"Boo-Boo Repair Specialist",
+		"Magos",
+	)
+
+/datum/job/head_of_personnel/New()
+	.=..()
+	alt_titles += list(
+		"Captain't",
+		"Chief Navigational Officer",
+		"First Officer",
+		"First Mate",
+		"Second Officer",
+		"Second Mate",
+		"Staff Manager",
+		"Staff Overseer",
+	)
+
+/datum/job/nanotrasen_consultant/New()
+	.=..()
+	alt_titles += list(
+		"Nanotrasen Representative",
+		"Bridge Advisor",
+		"Bridge Consultant",
+		"Bridge Diplomat",
+		"Bridge Inspector",
+		"Bridge Officer",
+		"Bridge Snitch",
+		"Central Advisor",
+		"Central Command Representative",
+		"Central Consultant",
+		"Central Diplomat",
+		"Central Inspector",
+		"Central Officer",
+		"Chief Safety Officer",
+		"Command Advisor",
+		"Command Consultant",
+		"Command Diplomat",
+		"Command Inspector",
+		"Command Liaison",
+		"Command Officer",
+		"Compliance Officer",
+		"Conduct Officer",
+		"Corporate Assistant",
+		"Corporate Advisor",
+		"Corporate Butler",
+		"Corporate Glowie",
+		"Corporate Inspector",
+		"Corporate Lawyer",
+		"Corporate Liaison",
+		"Corporate Maid",
+		"Corporate Snitch",
+		"Corporate Officer",
+		"Corporate Representative",
+		"Corporate Servant",
+		"Courtesan",
+		"Fax Monkey",
```

Comment:
no

Reply:
Taken out in an upcoming commit, no fax monkey :(